src/command/setup.rs

```diff
@@ -43,6 +43,16 @@ mod os {
         let home = volta_home()?;
         let formatted_home = format_home(home.root());
 
+        // Don't update the user's shell config files if VOLTA_HOME and PATH already contain what we need.
+        let home_in_path = match env::var_os("PATH") {
+            Some(paths) => env::split_paths(&paths).find(|p| p == home.root()),
```

Comment:
I believe this check will almost always fail, since we don't add `~/.volta` to the PATH, but rather `~/.volta/bin`.

```suggestion
            Some(paths) => env::split_paths(&paths).find(|p| p == home.shim_dir()),
```

Reply:
Sorry to keep going round-and-round, but it looks like we may have lost the `shim_dir` change with the follow-up that added the `debug!` statement below.